lizmap/modules/lizmap/controllers/edition.classic.php

```diff
@@ -961,8 +961,15 @@ public function saveFormDataToDb($form){
               $value = 'NULL';
             break;
           case 'text':
+            if ( !$value or strlen(strtolower($value)) == 0)
```

Comment:
why strtolower ?

If $value='0', here 'null' will be stored. Are you sure you want this result?

Reply:
if we have `'null'` it's because the user has wrote `null` in the HTML form isn't it ?

Reply:
if the user enter `0`, the result of !$value is true, so 'null' will be stored.

Reply:
so we have to use is_null ?